lib/sqlalchemy/sql/elements.py

```diff
@@ -2223,13 +2264,13 @@ def _select_iterable(self) -> _SelectIterable:
     _allow_label_resolve = False
 
     @property
-    def _is_star(self):
+    def _is_star(self) -> bool:
```

Comment:
Mypy doesn't like overriding writable attribute with read-only property. My first thought was that it's mypy's general [weirdness](https://github.com/python/mypy/issues/12440) with `@property`, or [a bug](https://github.com/python/mypy/issues/4125) - but looks like this has been fixed. [Here](https://stackoverflow.com/questions/58349417/how-to-annotate-attribute-that-can-be-implemented-as-property) a guy makes a point about how this might become a legitimate problem. Anyway, not sure how to proceed, and this isn't the only case.